.gitmodules

```diff
@@ -22,3 +22,9 @@
 [submodule "projects/1000GenomesProject"]
   path = projects/1000GenomesProject
   url = https://github.com/conpdatasets/1000GenomesProject.git
+[submodule "projects/conp-dataset-1KGP"]
+  path = projects/conp-dataset-1KGP
+  url = https://github.com/emmetaobrien/conp-dataset-1KGP
+[submodule "projects/conp-dataset-refseq"]
+	path = projects/conp-dataset-refseq
```

Comment:
Maybe change the name to `refseq` instead of `conp-dataset-refseq`? The url is ok.

Reply:
This is now done.